go/client/chat_svc_handler.go

```diff
@@ -818,14 +826,20 @@ func (c *chatServiceHandler) makePostHeader(ctx context.Context, arg sendArgV1,
 	default:
 		return nil, fmt.Errorf("multiple conversations matched")
 	}
+	var ephemeralMetadata chat1.MsgEphemeralMetadata
+	if arg.ephemeralLifetime.Duration != 0 {
```

Comment:
Why even bother checking for 0 here? That is what is getting set anyway if you just let it through.